docs/.vuepress/navbar.js

```diff
@@ -0,0 +1,23 @@
+const navbar_en = [
+  {
+    text: "Docs",
```

Comment:
```suggestion
    text: "Docs",
    link: "/docs/"
```

Shoudln't need to link the other index.md's either. These all render to their respective `/dir_name/` so the index is uncesscary.

Reply:
Thats good to know, I'll go ahead and commit, what you have here.